Makefile

```diff
@@ -30,5 +30,8 @@ docs/index.html: pydeps
 docs/preview-bundle.js: jsdeps
 	./node_modules/.bin/browserify -d -t [ reactify --es6 ] reactify-components.jsx -o docs/preview-bundle.js
 
+watch-preview: jsdeps
```

Comment:
I added this in as I found it to be super convenient while doing development. I can remove it if we don't want it in this diff.